src/sentry/static/sentry/app/views/organizationIntegrations/integrationProviderRow.tsx

```diff
@@ -0,0 +1,197 @@
+import {withTheme} from 'emotion-theming';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {openIntegrationDetails} from 'app/actionCreators/modal';
+import {PanelItem} from 'app/components/panels';
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import CircleIndicator from 'app/components/circleIndicator';
+import InstalledIntegration, {
+  Props as InstalledIntegrationProps,
+} from 'app/views/organizationIntegrations/installedIntegration';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {growDown, highlight} from 'app/styles/animations';
+import {IntegrationProvider, Integration} from 'app/types';
+
+type Props = {
+  provider: IntegrationProvider;
+  orgId: string;
+  onInstall: (integration: Integration) => void;
+  onRemove: (integration: Integration) => void;
+  onDisable: (integration: Integration) => void;
+  onReinstall: (integration: Integration) => void;
+  enabledPlugins: string[];
+  newlyInstalledIntegrationId: string;
+  integrations: Integration[];
+};
+
+export default class ProviderRow extends React.Component<Props> {
+  static propTypes = {
+    provider: PropTypes.object.isRequired,
+    integrations: PropTypes.array.isRequired,
+    orgId: PropTypes.string.isRequired,
+    onInstall: PropTypes.func.isRequired,
+    onRemove: PropTypes.func.isRequired,
+    onDisable: PropTypes.func.isRequired,
+    onReinstall: PropTypes.func.isRequired,
+    enabledPlugins: PropTypes.array,
+    newlyInstalledIntegrationId: PropTypes.string,
+  };
+
+  static contextTypes = {
+    organization: SentryTypes.Organization,
+  };
+
+  static defaultProps = {
+    enabledPlugins: [],
```

Comment:
@NisanthanNanthakumar Let's get rid of anything related to plugins in this component